modules/README.md

```diff
@@ -20,22 +20,26 @@ Modules labeled with the ![community-badge] badge are contributed by
 the community (including the HPC Toolkit team, partners, etc.). Community modules
 are located in the [community folder](../community/modules/README.md).
 
+Modules labeled with the ![deprecated-badge] badge are now deprecated and may be
+removed in the future. Customers are advised to transition to alternatives.
+
 Modules that are still in development and less stable are labeled with the
 ![experimental-badge] badge.
 
 [core-badge]: https://img.shields.io/badge/-core-blue?style=plastic
 [community-badge]: https://img.shields.io/badge/-community-%23b8def4?style=plastic
 [stable-badge]: https://img.shields.io/badge/-stable-lightgrey?style=plastic
 [experimental-badge]: https://img.shields.io/badge/-experimental-%23febfa2?style=plastic
+[deprecated-badge]: https://img.shields.io/badge/-deprecated-%23fea2a2?style=plastic
 
 ### Compute
 
 * **[vm-instance]** ![core-badge] : Creates one or more VM instances.
-* **[SchedMD-slurm-on-gcp-partition]** ![community-badge] : Creates a partition
+* **[SchedMD-slurm-on-gcp-partition]** ![community-badge] ![deprecated-badge] : Creates a partition
```

Comment:
nit: Can you push the deprecated items to the bottom of the list.

Reply:
Good point. Addressed.